helm/default-apps-openstack/values.yaml

```
@@ -1,12 +1,12 @@
 name: default-apps-openstack
 serviceType: managed
 
-project:
-  branch: "[[ .Branch ]]"
-  commit: "[[ .SHA ]]"
+cluster:
+  name: ""
+  namespace: ""
+  organization: ""
 
-image:
-  registry: quay.io
-  name: giantswarm/default-apps-openstack
-  tag: v0.0.1
-  pullPolicy: IfNotPresent
+remoteNamespace: kube-system
```

```suggestion
remoteNamespace: giantswarm
```

Pretty sure this is the correct namespace.

---

I'm not sure it makes sense to have a value for this. For apps like coredns we expect them to be in kube-system. If we change this I think things will break.